.github/workflows/build.yaml

```diff
@@ -5,20 +5,18 @@ jobs:
   ros2:
     runs-on: ubuntu-20.04
     container:
-      image: docker://ros:foxy-ros-base-focal
+      image: docker://ros:galactic-ros-base-focal
```

Comment:
Can we open a separate PR to merge the CI fixes ASAP? I don't think it should be part of this PR.